assists/y/v.sh

```diff
@@ -1,7 +1,8 @@
-#!/usr/bin/env bash
+#!/bin/dash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
Please leave the hashbang as is. It's an inter-os compatibility thing here.

Reply:
The main change is switching the shell to dash because it is a faster shell and the script works fine using dash. Would `#!/usr/bin/env dash` be okay or do you want it to remain as bash?

Reply:
@con-f-use do you know if I have to @ you for you to get notified of my review comments? I dont want to pointlessly @ you but I am not sure that you will know that I replied to you otherwise.

Reply:
You don't have to @ me, I am auto-subscribed to items I comment out.

The difference in startup time between bash and dash is negligible, especially if systemd and a browser runs on the system and bash is still the more common shell. If you change the hashbangs here, that means I have to patch them in instantNIX.

Reply:
Thanks for clarifying.

I will change it back to bash then. I changed it because some other assists used it as well as instantassist itself and I though it would make sense to use, but I get that the performance difference is not that big. I do have a few questions though.
1. some scripts use dash already with `#!/usr/bin/dash`. Does this cause any problems with packaging? would `#/usr/bin/env dash` cause any problems.
2. More bash assists seem to use `#!/bin/bash` than `#!/usr/bin/env bash`. is there any reason for these differences or does it just not matter.